packages/block-editor/src/components/contrast-checker/index.native.js

```diff
@@ -0,0 +1,99 @@
+/**
+ * External dependencies
+ */
+import { Text } from 'react-native';
+import tinycolor from 'tinycolor2';
+
+/**
+ * WordPress dependencies
+ */
+import { speak } from '@wordpress/a11y';
+import { __ } from '@wordpress/i18n';
+import { useEffect } from '@wordpress/element';
+import { withPreferredColorScheme } from '@wordpress/compose';
```

Comment:
Since this is a functional component we could use `usePreferredColorSchemeStyle` instead of `withPreferredColorScheme` to avoid using a HOC. Then we can just replace `getStylesFromColorScheme(...)` to `usePreferredColorSchemeStyle(...)`

Reply:
I went ahead to implement this in https://github.com/WordPress/gutenberg/pull/34902/commits/0630f8913478ad337e724cddef592bbeaa98aece, thanks!